youtube_dl/extractor/dailymotion.py

```diff
@@ -182,13 +182,18 @@ def _real_extract(self, url):
                 if metadata_url:
                     metadata_url = metadata_url.replace(':videoId', video_id)
                 else:
-                    metadata_url = update_url_query(
-                        'https://www.dailymotion.com/player/metadata/video/%s'
-                        % video_id, {
-                            'embedder': url,
-                            'integration': 'inline',
-                            'GK_PV5_NEON': '1',
-                        })
+                    if (17 < age_limit) :
```

Comment:
Please continue to make the code [PEP8](https://www.python.org/dev/peps/pep-0008/) compliant, which the extra space between the condition and the `:` violate, as does the extra space between your `else` and `:` below

The parens around the test are also unnecessary, but I don't believe they're strictly speaking wrong